openmrs/apps/clinical/dashboard.json

```diff
@@ -8,9 +8,9 @@
                 "translationKey":"DASHBOARD_TITLE_PATIENT_INFORMATION_KEY",
                 "type":"patientInformation",
                 "displayOrder":0,
-                "ageLimit":100,
-                "patientAttributes":[  
-
+                "ageLimit":12,
+                "showDOB":false,
```

Comment:
`"showDOB": false`
Did we get this requirement for TLC, ie, remove the date of birth ?

`"ageLimit":12,`
I think this affects the whole extension. So that means you are saying that the whole widget shouldn't appear when patient is older than 12... ? Also while trying, I think it simply does not work. My patient is 15 but the widget still appears. Have you been able to validate that it works?

Could this widget be simply be removed. It does not bring anything that is not already on the header right?